funtofem/optimization/openmdao_component.py

```diff
@@ -139,12 +144,11 @@ def compute(self, inputs, outputs):
         track_history = self.options["track_history"]
         model = driver.model
         design_out_file = self.options["design_out_file"]
-        new_design = self.update_design(inputs, analysis=True)
+        self.update_design(inputs, analysis=True)
 
-        if new_design:
```

Comment:
Why aren't we still checking if it's a new design?

Reply:
See below comment, I will make an issue for this.